lib/std/enums.zig

```diff
@@ -1505,3 +1505,70 @@ test values {
     };
     try testing.expectEqualSlices(E, &.{ .X, .Y, .Z }, values(E));
 }
+
+/// Returns an enum containing the combined elements of the given enum types.
+/// The values are incremental integers starting at 0.
+fn CombinedEnums(comptime enum_types: anytype) type {
```

Comment:
```suggestion
fn Combined(comptime enum_types: anytype) type {
```

[Avoid Redundant Names in Fully-Qualified Namespaces](https://ziglang.org/documentation/master/#Avoid-Redundant-Names-in-Fully-Qualified-Namespaces)